modules/microshift-cni.adoc

```diff
@@ -0,0 +1,271 @@
+<2> Default value is an empty string, which means disabled.
+
+To customize your configuration, use the following table for reference to valid values that can be used in your `ovn.yaml` config file.
+
+.Supported optional OVN-Kubernetes configurations for {product-title}.
+
+[cols="5",options="header"]
+|===
+|Field
+|Type
+|Default
+|Description
+|Example
+
+|`ovsInit.disableOVSInit`
+|bool
+|false
+|Skip configuring OVS bridge `br-ex` in `microshift-ovs-init.service`
+|true <1>
+
+|`ovsInit.gatewayInterface`
+|Alpha
+|eth0
+|Ingress that is the API gateway
+|eth0
+
+|`ovsInit.externalGatewayInterface`
+|Alpha
+|eth1
+|Ingress routing external traffic to your services and pods inside the node
+|eth1
+
+|mtu
+|uint32
+|1400
+|MTU value used for the pods
+|1300
+|===
+
+<1> The OVS bridge is required. When `disableOVSInit` is true, OVS bridge `br-ex` must be configured manually.
+
+.Example `ovn.yaml` config file:
+[source, yaml]
+----
+ovsInit:
+  disableOVSInit: true
+  gatewayInterface: eth0
+  externalGatewayInterface: eth1
+mtu: 1300
+----
+
+[IMPORTANT]
+When `disableOVSInit` is set to true in the `ovn.yaml` config file, the OVS bridge br-ex must be manually configured.
+
+[id="microshift-bridge-mapping_{context}"]
+== Bridge mappings
+Bridge mappings allow provider network traffic to reach the physical network. Traffic leaves the provider network and arrives at the `br-int` bridge. A patch port between `br-int` and `br-ex` then allows the traffic to traverse to and from the provider network and the edge network. Kubernetes pods are connected to the `br-int` bridge via virtual ethernet pair: one end of the virtual ethernet pair is attached to the pod namespace, the other end is attached to the `br-int` bridge.
+
+=== Primary gateway interface
+You can specify the desired host interface name in the `ovn.yaml` config file as `gatewayInterface`. The specified interface will be added in OVS bridge br-ex which acts as gateway bridge for the CNI network.
+
+=== Secondary gateway interface
+You can set up one additional host interface for cluster ingress and egress in the `ovn.yaml` config file.
+The additional interface will be added in a second OVS bridge `br-ex1`. Cluster pod traffic directed to the additional host subnet will be routed through br-ex1.
+//Q: do we want to specify which interface the traffic leaves the provider network from?
+Either two or three OVS bridges are created based on the CNI configuration:
+
+Default deployment::
+* The externalGatewayInterface in not specified in the `ovs.yaml` config file.
+* Two OVS bridges, `br-ex` and `br-int`, are created.
+
+Customized deployment::
+* The externalGatewayInterface is user-specified in the `ovs.yaml` config file.
+* Three OVS bridges will be created: `br-ex`, `br-ex1` and `br-int`.
+
+//Q: We need to include the third bridge if we want to include this level of detail.
```

Comment:
I'd simply say br-ex, br-ex1, br-int, and avoid going into too much inner detail may be?

Keeping it would be fine too I think. mixed feelings here :)